xmpp/src/test/java/ch/qos/logback/classic/net/XmppAppenderTest.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) 2016, The logback-contrib developers. All rights reserved.
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation
+ *
+ *   or (per the licensee's choosing)
+ *
+ * under the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation.
+ */
+package ch.qos.logback.classic.net;
+
+import org.junit.After;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Ignore
+public class XmppAppenderTest {
+
+    @Test 
+    public void testSimpleMessage() {
```

Comment:
As you can see that test is ignored on class level.
This is because it requires an external resource (running XMPP server) so it's impossible to do real unit test for this appender.

I can remove that test if you want.